src/test/java/dev/t3d/happraisal/service/InterviewServiceTest.java

```diff
@@ -0,0 +1,99 @@
+package dev.t3d.happraisal.service;
+
+import dev.t3d.happraisal.entity.Form;
+import dev.t3d.happraisal.entity.Interview;
+import dev.t3d.happraisal.entity.Person;
+import dev.t3d.happraisal.entity.QuestionAnswer;
+import dev.t3d.happraisal.repository.InterviewRepository;
+import org.assertj.core.api.BDDAssertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.BDDMockito;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class InterviewServiceTest {
+
+  @Mock private InterviewRepository interviewRepository;
+
+  @InjectMocks private InterviewService interviewService;
+
+  @Test
+  void should_create_interview() {
+    // given
+    final Interview expectedInterviewToSave = getInterview();
+
+    // when
+    interviewService.create(expectedInterviewToSave);
+
+    // then
+    var interviewArgumentCaptor = ArgumentCaptor.forClass(Interview.class);
+    BDDMockito.verify(interviewRepository).save(interviewArgumentCaptor.capture());
+    var actualInterviewToSave = interviewArgumentCaptor.getValue();
+    BDDAssertions.then(actualInterviewToSave)
+        .as("Check is an interview has been save")
+        .isEqualTo(expectedInterviewToSave);
+  }
+
+  @Test
+  void should_get_interview_by_id() {
+    // given
+    final Interview expectedInterview = getInterview();
+    var interviewId = UUID.fromString("149e7f49-e66e-4cfc-8485-37993311d9e2");
+    BDDMockito.given(interviewRepository.getById(interviewId)).willReturn(expectedInterview);
+
+    // when
+    final Interview actualInterview = interviewService.getById(interviewId);
+
+    // then
+    BDDAssertions.then(actualInterview)
+        .as("Check is an interview has been return by its id")
```

Comment:
"Check is an interview has been return by its id"
should be
"Check if an interview has been returned by its id"